src/TypeScript/Rules/ActiveRulesCalculator.cs

```diff
@@ -89,7 +89,9 @@ private static Rule Convert(RuleDefinition ruleDefinition)
                 Key = ruleDefinition.EslintKey,
 
                 // TODO: handle parameterised rules #2284
-                Configurations = ruleDefinition.DefaultParams
+                Configurations = ruleDefinition.DefaultParams,
+
+                FileTypeTarget = new []{"MAIN"}
```

Comment:
Categorising the dozen `TEST` rules as `MAIN` rules have presumably means they will be run against all files. If they do report any issues then they will be false positives.

We should check whether those rules will report issues against `MAIN` files, If they do, we should consider excluding them from the rules config (e.g. by having a hard-coded list of rules we don't want to run ever).

I'll add a ticket to the board to investigate whether this is necessary.